db/sql/00_msar.sql

```diff
@@ -291,13 +222,13 @@ Args:
   rel_id:  The OID of the relation.
   col_id:  The attnum of the column in the relation.
 */
-SELECT quote_ident(attname::text) FROM pg_attribute WHERE attrelid=rel_id AND attnum=col_id;
+SELECT attname::text FROM pg_attribute WHERE attrelid=rel_id AND attnum=col_id;
 $$ LANGUAGE sql RETURNS NULL ON NULL INPUT;
 
 
 CREATE OR REPLACE FUNCTION
 msar.get_column_name(rel_id oid, col_name text) RETURNS text AS $$/*
-Return the name for a given column in a given relation (e.g., table).
+Return the UNQUOTED name for a given column in a given relation (e.g., table).
 
 More precisely, this function returns the quoted name of attributes of any relation appearing in the
```

Comment:
```suggestion
More precisely, this function returns the unquoted name of attributes of any relation appearing in the
```

Reply:
Thanks! Fixed in a15f6fefbf7d269f5039b16bfa9c1fbb85a63f4e